lib/manager/components/deployment/DeploymentViewer.js

```diff
@@ -251,6 +251,8 @@ export default class DeploymentViewer extends Component<Props, State> {
     const boundsTooLarge = east - west > BOUNDS_LIMIT || north - south > BOUNDS_LIMIT
     const bounds = this._getBounds()
 
+    // $FlowFixMe flow array method list is outdated
+    if (bounds.flat().includes('NaN')) return null
```

Comment:
Can you just add a comment explaining why you might get a NaN from this?

Reply:
Good call thanks